docs/_build/html/_sources/methods.rst.txt

```diff
@@ -0,0 +1,444 @@
+.. set up matlab code highlighting
+.. highlight:: matlab
+
+.. set up referencing
+.. _methods:
+
+Methods
+=======
+
+This documentation will focus on using ``xolotl`` in ``MATLAB``, though it is
+possible to compile and run the ``C++`` code on its own.
+
+A ``xolotl`` object can be instantiated ::
+
+  x = xolotl;
+
+For consistency, this documentation will assume a ``xolotl`` object named ``x``.
+
+.. hint::
+
+  You can list all the methods of ``xolotl`` object ``x`` by ::
+
+    x.methods
+
+.. _children:
+
+Children
+^^^^^^^^
+
+Returns the children of any ``cpplab`` object as a cell of character vectors.
+Children are also ``cpplab`` objects. For example, if ``HH`` is the only compartment in ``x`` then ::
+
+  x.Children
+
+returns ::
+
+  {'HH'}
+
+This method is inherited from the superclass ``cpplab``.
+
+.. _add:
+
+add
+^^^
+
+Adds a compartment, conductance, or synapse to the ``xolotl`` object. To add
```

Comment:
`cpplab` allows you to add any `cpplab` object to anything else, but `xolotl` assumes that you're doing sane things: if you add a compartment to a conductance or something silly, you will get an error.

Reply:
done